tfjs-layers/src/layers/preprocessing/random_width.ts

```diff
@@ -123,32 +117,31 @@ export class RandomWidth extends BaseRandomLayer {
     kwargs: Kwargs): Tensor[]|Tensor {
 
     return tidy(() => {
-        const input = getExactlyOneTensor(inputs);
-        const inputShape = input.shape;
-        this.imgHeight = inputShape[inputShape.length - 3];
-        const imgWidth = inputShape[inputShape.length - 2];
-
-        this.widthFactor = this.randomGenerator.randomUniform([1],
-          (1.0 + this.widthLower), (1.0 + this.widthUpper),
-          'float32', this.seed
-        );
-
-        this.seed = this.randomGenerator.next();
-
-        this.adjustedWidth = this.widthFactor.dataSync()[0] * imgWidth;
-        this.adjustedWidth = Math.round(this.adjustedWidth);
-
-        const size:[number, number] = [this.imgHeight, this.adjustedWidth];
-
-        switch (true) {
-          case this.interpolation === 'bilinear':
-            return image.resizeBilinear(inputs, size);
-          case this.interpolation === 'nearest':
-            return image.resizeNearestNeighbor(inputs, size);
-          default:
-            throw new Error(`Interpolation is ${this.interpolation}
-            but only ${[...INTERPOLATION_METHODS]} are supported`);
-        }
+      const input = getExactlyOneTensor(inputs);
+      this.imgHeight = input.shape[input.shape.length - 3];
+      const imgWidth = input.shape[input.shape.length - 2];
+
+      this.widthFactor = randomUniform([1],
+        (1.0 + this.widthLower), (1.0 + this.widthUpper),
+        'float32', this.randomGenerator.currentSeed
+      );
+
+      this.randomGenerator.next();
```

Comment:
`this.randomGenerator.next()` increments the seed and then returns the new seed, so you can use its return value instead of incrementing it separately.

```suggestion
        'float32', this.randomGenerator.next());
```

If you want to use the current seed before incrementing it, change the implementation of `RandomGenerator.next` to return `seed++` (increments after getting the value) instead of `++seed` (increments before getting the value).